src/services/redirectToDestination.js

```diff
@@ -0,0 +1,31 @@
+//expects purposeParams, based on the value redirects to respective destination
+
+export function redirectToDestination(purposeParams, userID, redirectID){
+    var redirectURL = "";
+    var fullurl = "";
+
+    switch(purposeParams){
+        case 'plio':
+            //constructs the URL based on the redirectTo param
+            redirectURL = process.env.VUE_APP_BASE_URL_PLIO;
+            var url = new URL(redirectURL + redirectID); //adds plioID to the base plio link
+            //adds params; api key and student SRN
+            var queryparams = new URLSearchParams({
+                                api_key: process.env.VUE_APP_AF_API_KEY,
+                                unique_id: userID,
+                                });
+            fullurl = url + "?" + queryparams;
+            break;
+
+        case 'liveclass':
+            //constructs the URL based on the redirectTo param
+            redirectURL = process.env.VUE_APP_BASE_URL_MEET;
+            fullurl = new URL(redirectURL + redirectID); //adds meetID to the base plio link
+            break;
+        
+        default:
+            alert('Destination is unsupported')
```

Comment:
Alert is wrong here. It needs to just return a 500 error, considering it's a GET request usually to the auth layer

Reply:
Alter will show it to the end user. Which is not useful to us

Reply:
Yes alert will make a popup on the browser